Open Judge System/Services/OJS.Services/Business/ParticipantScores/Models/ContestParticipationSummaryExcel.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+
+namespace OJS.Services.Business.ParticipantScores.Models
+{
+    using System.IO;
+    using System.Linq;
+    using MissingFeatures;
+    using NPOI.HSSF.UserModel;
+    using NPOI.SS.UserModel;
+    using System.Collections.Generic;
+    using OJS.Web.Common.Extensions;
+    using OJS.Common.Extensions;
+    
+    public class ContestParticipationSummaryExcel
+    {
+        public HSSFWorkbook Workbook { get; set; }
+        
+        public ContestParticipationSummary ContestParticipationSummary { get; set; }
```

Comment:
IMO it will be better to encapsulate the properties because currently the implementation is not doing null checks and the properties can be reassigned.